.gitignore

```diff
@@ -4,7 +4,9 @@
 tmp/
 
 # intellij files
-.idea
+.idea/*
+# Icon for JetBrains Toolbox
+!/.idea/icon.png
```

Comment:
The change is unnecessary, we can always use `git add -f [file]` to force let git track the file.